components/packaging/common/load-telemetry-plugin

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env sh
+
+BASE=/var/lib/docker
```

Comment:
This directory is configurable as well, no guarantee that's where docker stores its files